IFC4x3/Properties/t/TunnelInclination_2$VkG7ESn3rQ2WJSlegUBN/DocProperty.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This sounds a bit complicated. Geometric data described by enumerations. Albeit I have no idea what added value this brings, if it is a requirement, then it is a requirement. 😄

Reply:
Agreed to keep.